_dev/apps/ui/src/store/modules/product-feed/actions.ts

```diff
@@ -318,7 +318,7 @@ export default {
     return result;
   },
   async [ActionsTypes.REQUEST_SYNCHRONISATION]({rootState}: Context, full = false) {
-    const response = await fetch(`https://eventbus-sync.psessentials.net/sync/trigger${full ? '-full' : ''}`, {
+    const response = await fetch(`https://api.cloudsync.prestashop.com/sync/v1//trigger${full ? '-full' : ''}`, {
       method: 'GET',
```

Comment:
Shall we change the method as well? I heard `GET` were deprecated

Reply:
Very right you are. Fixing the code I will.